pkg/envtest/server.go

```diff
@@ -256,3 +253,11 @@ func (te *Environment) defaultTimeouts() error {
 	}
 	return nil
 }
+
+func (te *Environment) useExistingCluster() bool {
+	if te.UseExistingCluster == nil {
+		useExistingCluster := strings.ToLower(os.Getenv(envUseExistingCluster)) == "true"
+		te.UseExistingCluster = &useExistingCluster
```

Comment:
no need to populate this here -- just return the env var without populating

Reply:
comment addressed, ptal